zipline/data/us_equity_pricing.py

```diff
@@ -204,9 +189,34 @@ class BcolzDailyBarWriter(object):
         'volume': float64,
     }
 
-    def __init__(self, filename, sessions, calendar):
+    def __init__(self, filename, calendar, start_session, end_session):
+        """
```

Comment:
Why did we move this docstring?  The convention pretty much everywhere I've seen is to put the docs for `__init__` at the top of the class.